### PR TITLE
Update FSharp.Data

### PR DESCRIPTION
**Pull request by ovatsus**

### CODE REVIEW

File: _data/projects.yml [hunk 1]
```diff
@@ -127,9 +127,9 @@
     link: https://github.com/mexx/FeatureSwitcher/issues?labels=up-for-grabs&state=open
 
 - name: FSharp.Data
-  desc:
+  desc: "The F# Data library implements type providers for working with structured file formats (CSV, JSON and XML) and for accessing the WorldBank and Freebase data. It also includes helpers for parsing JSON and CSV files and for sending HTTP requests."
   site: https://github.com/fsharp/FSharp.Data
-  tags:
+  tags: [fsharp data typeprovider WorldBank Freebase CSV XML JSON HTTP]
```
Comment (shiftkey) [inline _data/projects.yml]: We've been using `,` elsewhere to separate these tags values - I'm not sure if Jekyll is smart enough to parse these. 

Will check back and confirm.

Comment (dahlbyk) [inline _data/projects.yml]: Also, let's use `F#` instead of `fsharp`.

**Comment by ovatsus** [comment]
Updated

File: _data/projects.yml [hunk 2]
```diff
@@ -127,9 +127,9 @@
     link: https://github.com/mexx/FeatureSwitcher/issues?labels=up-for-grabs&state=open
 
 - name: FSharp.Data
-  desc:
+  desc: "The F# Data library implements type providers for working with structured file formats (CSV, JSON and XML) and for accessing the WorldBank and Freebase data. It also includes helpers for parsing JSON and CSV files and for sending HTTP requests."
   site: https://github.com/fsharp/FSharp.Data
-  tags:
+  tags: ["F#", data, typeprovider, WorldBank, Freebase, CSV, XML, JSON, HTTP]
```
Comment (shiftkey) [inline _data/projects.yml]: Just F#, drop the braces and this will be :sparkles:

Comment (ovatsus) [inline _data/projects.yml]: `#` is a comment in yaml, `F#` has to go within quotes or it will break, no?

Comment (shiftkey) [inline _data/projects.yml]: Nope, see the FAKE project has it like that

https://github.com/dahlbyk/up-for-grabs.net/blob/gh-pages/_data/projects.yml#L116

Comment (ovatsus) [inline _data/projects.yml]: Done

**Comment by shiftkey** [comment]
Thanks!

**Pull request merged**